### PR TITLE
[INLONG-9977][Audit] Audit-service increases the capabilities of openapi

### LINKED ISSUE #9977: [Feature][Audit] Audit-service increases the capabilities of openapi
### Description

In order to improve the concurrency and decoupling of audit external services, audit-service needs to provide a complete set of external openapi.
image

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by doleyzi**
### Prepare a Pull Request

- Fixes #9977

### Motivation
image

*In order to improve the concurrency and decoupling of audit external services, audit-service needs to provide a complete set of external openapi.*

### Modifications

*By adding an http interface to provide external audit data, there is less coupling with other modules..*

### CODE REVIEW

**Review by luchunliang — DISMISSED** [review]

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/entities/JdbcConfig.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class JdbcConfig {
+
+    String driverClass;
+    String JdbcUrl;
+    String userName;
+    String passWord;
```
Comment (dockerzhang) [inline inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/entities/JdbcConfig.java:30]:
```suggestion
    String password;
```

File: inlong-audit/pom.xml [hunk 2]
```diff
@@ -37,6 +37,7 @@
         <module>audit-sdk</module>
         <module>audit-release</module>
         <module>audit-docker</module>
+        <module>audit-service</module>
```
Comment (dockerzhang) [inline inlong-audit/pom.xml:40]: add `audit-docker` to the end.

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/cache/AbstractCache.java [hunk 3]
```diff
@@ -82,7 +83,11 @@ public Cache<String, StatData> getCache() {
      * @return
      */
     public List<StatData> getData(String key) {
-        return Arrays.asList(cache.getIfPresent(key));
+        StatData statData = cache.getIfPresent(key);
+        if (null == statData) {
+            return new LinkedList<>();
+        }
+        return Arrays.asList(statData);
```
Comment (aloyszhang) [inline inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/cache/AbstractCache.java:90]:
```suggestion
         return Collections.singletonList(statData);
```

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/service/ApiService.java [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+                JsonObject responseJson = new JsonObject();
+
+                Map<String, String> params = parseRequestURI(exchange.getRequestURI().getQuery());
+                if (!checkParams(params)) {
+                    handleInvalidParams(responseJson, exchange);
+                } else {
+                    handleLegalParams(responseJson, params);
+                }
+
+                byte[] bytes = responseJson.toString().getBytes();
```
Comment (justinwwhuang) [inline inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/service/ApiService.java:134]: Do we need to specify the encoding format here?

**Review by justinwwhuang — APPROVED** [review]

**Review by justinwwhuang — APPROVED** [review]

File: inlong-audit/bin/service-stop.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# this program kills the audit store
+ps -ef |grep "audit-service" | grep -v grep|awk '{print $2}' | xargs kill -9
```
Comment (aloyszhang) [inline inlong-audit/bin/service-stop.sh:23]: shutdown service gracefully is preferred than forcefully(kill =9)

**Review by justinwwhuang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**